index.html

```diff
@@ -0,0 +1,158 @@
+        <div class="programs" id="lecture">
+          <img src="./Images/lecture-logo.png" alt="lecture-logo" />
+          <p class="program-names">Lecture</p>
+          <p class="program-details">
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit.
+            Reprehenderit, sequi.
+          </p>
+        </div>
+        <div class="programs" id="exhibiton">
+          <img src="./Images/exhibition-logo.png" alt="exhibition-logo" />
+          <p class="program-names">Exhibition</p>
+          <p class="program-details">
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quos
+            possimus earum nemo!
+          </p>
+        </div>
+        <div class="programs" id="forum">
+          <img src="./Images/forum-logo.png" alt="forum-logo" />
+          <p class="program-names">Forum</p>
+          <p class="program-details">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          </p>
+        </div>
+        <div class="programs" id="workshop">
+          <img src="./Images/workshop-logo.png" alt="workshop-logo" />
+          <p class="program-names">Workshop</p>
+          <p class="program-details">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Pariatur
+            provident ullam, quos nulla dolores cumque!
+          </p>
+        </div>
+        <div class="programs" id="network">
+          <img src="./Images/network-logo.png" alt="network-logo" />
+          <p class="program-names">Network</p>
+          <p class="program-details">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Et animi
+            nihil recusandae officiis consequatur nobis esse voluptates dicta
+            corporis corrupti.
+          </p>
+        </div>
+      </div>
+      <div>
+        <p id="join-link">Join Anime Fan Expo SF 2023</p>
+        <p id="see-program">See The Whole Program</p>
+      </div>
+    </section>
+    <section class="speakers">
+      <div id="feat-speakers">
+        <h3>Featured Speakers</h3>
+        <hr />
+      </div>
+
+      <!-- Speakers -->
+      <div class="speakers-column"></div>
+    </section>
+
+    <section class="partners-desktop">
+      <div id="partners-desktop">
+        <h3>Partners</h3>
+        <hr />
+      </div>
+      <div class="logos">
+        <img src="./Images/Marvel-logo.png" alt="marvel-logo" />
+        <img src="./Images/DC-logo.svg" alt="dc-logo" />
+        <img src="./Images/Toei_Animation_logo.png" alt="toei-logo" />
+        <img src="./Images/Studio_Pierrot-logo.webp" alt="pierrot-logo" />
+        <img src="./Images/studio-ghibli-logo.jpeg" alt="ghibli-logo" />
+      </div>
+    </section>
+    <footer class="footer-desktop">
+      <div id="footer-logo">
+        <img src="./Images/Logo.png" alt="logo" id="footer-img" />
+      </div>
+      <div id="footer-info">
+        <p>Lorem ipsum dolor sit amet consectetur</p>
+      </div>
+    </footer>
```

Comment:
- [ ] It is required to stick to the guiding design as close as possible. This includes, font-type, font-size, font-family, color, background-color, margin and padding among others. The margin and padding of your partner and footer sections are not in line with what we have on the guiding template. Kindly give a closer look at the guiding design and walk your design alongside.
- [ ] Also, there is horizontal scroll at the bottom of your page. Kindly correct that too.

### Your Design (Desktop)
![your-home-footer-spacing](https://user-images.githubusercontent.com/26597432/212050877-0f08b320-4d34-4f84-b394-6c88e7c908c6.png)

### Template (Desktop)
![temp-home-footer-spacing](https://user-images.githubusercontent.com/26597432/212050914-7b8a692c-6754-44f1-b269-008c404aae4f.png)